packages/foundry/.env.example

```diff
@@ -0,0 +1,21 @@
+# Template for foundry environment variables.
+
+# For local development, copy this file, rename it to .env, and fill in the values.
+
+# We provide default values so developers can start prototyping out of the box,
+# but we recommend getting your own API Keys for Production Apps.
+
+# DEPLOYER_PRIVATE_KEY is used while deploying contract, while deploying contract to
+# to anvil chain the value of it can be empty and we will use the prefunded account
+# which comes with anvil chain to deploy contract.
+# NOTE: You don't need to manage the value of DEPLOYER_PRIVATE_KEY manually, it should
+# be auto filled when run `yarn generate`.
+# Although `.env` is ignored by git, it's still important that you don't paste your
+# actual account private key and use the generated one.
+DEPLOYER_PRIVATE_KEY=
```

Comment:
Lol not sure if its good, maybe we cut it a bit?

Reply:
I think is fine, specially if we'd be switching soon to a Keystore implementation

Not sure if we should include it in this PR. What do you think?

Reply:
> Not sure if we should include it in this PR. What do you think?

Like we already have #813 PR tackling it, but I am not completely confident if we should cleanup and get those changes in this PR. Because in #813 I see it requires changes to core well tested stuff like `DeployHelpers.s.sol` and also `yarn deploy` is handle by node script completely. Not saying its bad at all and might be a good way to go but feels like require a fresh pair of eyes and bit of thinking since it touches DX a bit. 

In this PR wanted to get foundry working properly as it is similar to `create-eth`(since its tested nicely their) and then we can add / modify stuff to improve DX and brainstorm bit more on what things to improve.

Reply:
Sounds good! We can merge this PR first and then find a better way for #813 

Maybe we can wait to announce the "Foundry switch" until it's implemented.